src/dpl/src/CheckPlacement.cpp

```diff
@@ -20,17 +20,18 @@ using utl::DPL;
 
 using utl::format_as;
 
-void Opendp::checkPlacement(const bool verbose, const string& report_file_name)
+void Opendp::checkPlacement(const bool verbose,
+                            const std::string& report_file_name)
 {
   importDb();
 
-  vector<Cell*> placed_failures;
-  vector<Cell*> in_rows_failures;
-  vector<Cell*> overlap_failures;
-  vector<Cell*> one_site_gap_failures;
-  vector<Cell*> site_align_failures;
-  vector<Cell*> region_placement_failures;
-  vector<Cell*> edge_spacing_failures;
+  std::vector<Cell*> placed_failures;
+  std::vector<Cell*> in_rows_failures;
+  std::vector<Cell*> overlap_failures;
+  std::vector<Cell*> one_site_gap_failures;
+  std::vector<Cell*> site_align_failures;
+  std::vector<Cell*> region_placement_failures;
+  std::vector<Cell*> edge_spacing_failures;
```

Comment:
From https://google.github.io/styleguide/cppguide.html#Aliases

> However, local convenience aliases are fine in function definitions, private sections of classes, explicitly marked internal namespaces, and in .cc files:
> 
> // In a .cc file
> using ::foo::Bar;
> 

So there is no need to make this transformation in the cpp files.  Just move the using statement to the relevant cpp(s).

Reply:
sure, one _could_ do that inside a function or compilation unit, but it is very uncommon for `std::` ... in particular since we also have a sta::Vector with an uppercase letter, so that would be doubly confusing.
For `std::` we should spell it out always.

Reply:
Fine but I wouldn't say that is true for other namespaces in future PRs.

Vector is a bad idea - inheritance from a class with a non-virtual dtor just asks for trouble.

Reply:
Other namespaces, yes that is more common to have `using` inside compilation units or functions.

Yeah, `sta::Vector` (and `sta::Map`) is ... yeah indeed, I fully agree.